easybuild/easyconfigs/l/lightning/lightning-2.2.0.post0-foss-2023a-PyTorch-2.1.2-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'lightning'
```

Comment:
@NPotvin please use uppercase `Lightning` here, as that seems to be the official name?